plexpy/notifiers.py

```diff
@@ -1574,3 +1614,224 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+
+class Scripts(object):
+
+    def __init__(self, **kwargs):
+        pass
+
+    def conf(self, options):
+            return cherrypy.config['config'].get('Scripts', options)
+
+    def updateLibrary(self):
+        # For uniformity reasons not removed
+        return
+
+    def test(self, subject, message, *args, **kwargs):
+        self.notify(subject, message, *args, **kwargs)
+        return
+
+    def list_scripts(self):
+        scriptdir = plexpy.CONFIG.SCRIPTS_FOLDER
+        scripts = {' ': ''}
+
+        if scriptdir and not os.path.exists(scriptdir):
+            os.makedirs(scriptdir)
+
+        for root, dirs, files in os.walk(scriptdir):
+            for f in files:
+                name, ext = os.path.splitext(f)
+                ext = ext[1:]
+                if ext in ('rb', 'pl', 'bat', 'py', 'sh', 'cmd', 'php'):
+                    fp = os.path.join(root, f)
+                    scripts[fp] = fp
```

Comment:
I was sure the root had the full path of the file. But I might be mistaken. Any reason we should use real path and destroy and symbolic links?